.github/workflows/run-tests.yml

```diff
@@ -119,10 +117,6 @@ jobs:
         with:
           repository: "autoreduction/autoreduce-workspace"
 
-      - name: Pylint tests
-        run: |
-          pylint autoreduce_utils
```

Comment:
Do we not want to run pylint anymore?

Reply:
The way it was set up, pylint actually ran twice (once in the `Run autoreduction/autoreduce-actions/code_inspection@main` task and then again with the highlighted code here.

I believe this is because the autoreduce-actions/code-inspections role specified using pylint-django plugin with pylint, but utils doesn't use Django. I've since moved the selection of pylint-plugins to pyproject.toml so it can be selected per repo. 

All of this to say that pylint still runs, just once instead of twice.

Reply:
Ah! That is a great catch!